admin/index.html

```diff
@@ -1,25 +1,80 @@
-<!doctype html>
+<!DOCTYPE html>
 <html>
-<head>
-  <meta charset="utf-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>ADA Content Manager</title>
-</head>
-<body>
-   <!-- Include the script that builds the page and powers Netlify CMS -->
-  <script src="https://unpkg.com/netlify-cms@^2.0.0/dist/netlify-cms.js"></script>
-   <link href="/admin/config.yml" type="text/yaml" rel="cms-config-url">
-   <style>
-     img[alt="Logo"] {
-       margin-left: 75px;
-       width: 150px;
-     }
-     p {
-       color: red;
-     }
-     button {
-       margin-top: 0px !important;
-     }
-   </style>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>ADA Content Manager</title>
+  </head>
+  <body>
+    <!-- Include the script that builds the page and powers Netlify CMS -->
+    <script src="https://unpkg.com/netlify-cms@^2.0.0/dist/netlify-cms.js"></script>
+    <script>
```

Comment:
this script block is kind of complicated - can it go in its own file?

Reply:
You mean the netlify cms package?

Reply:
this comment was in reference to the css fetching code, which is gone now, so nevermind!